ci/manifest/logsearch-for-cloudfoundry-manifest.yml

```diff
@@ -68,11 +68,15 @@ jobs:
     elasticsearch:
       host: (( grab jobs.elasticsearch_master.networks.default.static_ips.[0] ))
       port: 9200
+    cloudfoundry:
+      firehose_events: (( grab properties.cloudfoundry.firehose_events ))  
     kibana_objects:
-      upload_data_files:
-        - /var/vcap/packages/logsearch-config-kibana-objects/kibana-objects-bulk-json
+      is_to_upload_defaults: true 
```

Comment:
@hannayurkevich could you please change the name of property `is_to_upload_defaults`, I think the name of this property not fully describes meaning without reference to spec file.